sdk/schemaregistry/Microsoft.Azure.Data.SchemaRegistry.ApacheAvro/tests/Models/Employee.cs

```diff
@@ -0,0 +1,70 @@
+// ------------------------------------------------------------------------------
+// <auto-generated>
+//    Generated by ApacheAvroTestTool, version 1.10.0.0
+//    Changes to this file may cause incorrect behavior and will be lost if code
+//    is regenerated
+// </auto-generated>
+// ------------------------------------------------------------------------------
+namespace TestSchema
+{
+	using System;
+	using System.Collections.Generic;
+	using System.Text;
+	using Avro;
+	using Avro.Specific;
+	
+	public partial class Employee : ISpecificRecord
+	{
+		public static Schema _SCHEMA = Avro.Schema.Parse("{\"type\":\"record\",\"name\":\"Employee\",\"namespace\":\"TestSchema\",\"fields\":[{\"name\":\"Na" +
+				"me\",\"type\":\"string\"},{\"name\":\"Age\",\"type\":\"int\"}]}");
+		private string _Name;
+		private int _Age;
+		public virtual Schema Schema
+		{
+			get
```

Comment:
nit:  Maybe consider using expression bodies to condense things a bit?

Reply:
This is a generated file by Apache Avro (says at the top of the file). I don't edit this. It is used for testing.